src/components/tipRecords/FormatDate.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <time :datetime="wholeDateAndTime" :title="wholeDateAndTime" >{{formatDate}}</time>
+</template>
+
+<script>
+export default {
+  name: 'FormatDate',
+  props: ['dateTimestamp'],
+  computed: {
+    formatDate() {
+      const today = new Date();
+      const tipDate = new Date(this.dateTimestamp);
+      const isToday = (today.toDateString() === tipDate.toDateString());
+      return isToday ? tipDate.toLocaleTimeString('en-US', { hourCycle: 'h24' }) : tipDate.toLocaleDateString('en-US');
+    },
+    wholeDateAndTime() {
+      return new Date(this.dateTimestamp).toLocaleString('en-US', { hourCycle: 'h24' });
```

Comment:
`datetime="YYYY-MM-DDThh:mm:ss"` should be machine readable which is Date().toISOString() while title should be human readable localeString of the full date